am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -37,13 +38,16 @@ void whenCreatingResourceAccessResourceAccessAppearsInDatabase() {
 
     @Test
     void whenCreatingResourceAccessMultipleEntriesAppearInDatabase() {
```

Comment:
In `GrantAccessIntegrationTest.java` there should be test showing that it is possible to grant access to multiple users. One test only.

Reply:
👍